test/e2e/app-dir/app-prefetch/prefetching.test.ts

```diff
@@ -220,7 +220,8 @@ describe('app dir - prefetching', () => {
     })
 
     const prefetchResponse = await response.text()
-    expect(prefetchResponse).not.toContain('Hello World')
```

Comment:
I tweaked this test because it didn't match the test description, and was failing (incorrectly) after making the changes in this PR.